site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -35,7 +35,7 @@ The default From Name used in sending emails when a from name is not provided on
 [field]#tenant.emailConfiguration.forgotPasswordEmailTemplateId# [type]#[UUID]# [optional]#Optional#::
 The Id of the Email Template that is used when a user is sent a forgot password email.
 
-[field]#tenant.emailConfiguration.host# [type]#[String]# [required]#Required# [since]#Available since 1.8.0#::
+[field]#tenant.emailConfiguration.host# [type]#[String]# [optional]#Optional# [default]#defaults to `localhost`# [since]#Available since 1.8.0#::
```

Comment:
I think these default values were added recently, perhaps in `1.28.0` - I'd have to check. We should probably add a comment that indicates prior to this version the field was required. 

For example:

> The host name of the SMTP server that FusionAuth will use for this tenant. 
> 
> Prior to version 1.28.0 this value was required. 

You'll want to check on the version the defaults were added. This would go for any "new" defaults added recently.